src/ride/ride.c

```diff
@@ -5790,6 +5846,12 @@ void game_command_set_ride_price(int *eax, int *ebx, int *ecx, int *edx, int *es
 	price = *edi;
 	secondary_price = (*edx >> 8);
 
+	if (ride_type == (rct_ride_type *)0xFFFFFFFF)
```

Comment:
If you have a moment can you refactor this function to change the variable name `ride_type` to `rideEntry` and to use `GET_RIDE_ENTRY` so that it matches the other functions.